### PR TITLE
set default value for cellline

### LINKED ISSUE #5: AltamISA parsing fails with template "stem_cell_core_bulk"
The following critical warnings are raised:

```
s_Study_Title.txt: Found annotated material/file without name: Material(type='Source Name', unique_name='p4-s0-Empty Source Name-1-1', name='', extract_label=None, characteristics=(Characteristics(name='Organism', value=[OntologyTermRef(name='Homo sapiens', accession='http://purl.bioontology.org/ontology/NCBITAXON/9606', ontology_name='NCBITAXON')], unit=None), Characteristics(name='Source Type', value=['hiPSC'], unit=None), Characteristics(name='Cellline', value=[''], unit=None), Characteristics(name='Genetic modfications', value=[''], unit=None)), comments=(), factor_values=(), material_type=None, headers=['Source Name', 'Characteristics[Organism]', 'Term Source REF', 'Term Accession Number', 'Characteristics[Source Type]', 'Characteristics[Cellline]', 'Characteristics[Genetic modfications]'])

s_Study_Title.txt: Found start node without original name: p4-s0-Empty Source Name-1-1

s_Study_Title.txt: Found annotated material/file without name: Material(type='Source Name', unique_name='p4-s0-Empty Source Name-1-2', name='', extract_label=None, characteristics=(Characteristics(name='Organism', value=[OntologyTermRef(name='Homo sapiens', accession='http://purl.bioontology.org/ontology/NCBITAXON/9606', ontology_name='NCBITAXON')], unit=None), Characteristics(name='Source Type', value=['hiPSC'], unit=None), Characteristics(name='Cellline', value=[''], unit=None), Characteristics(name='Genetic modfications', value=[''], unit=None)), comments=(), factor_values=(), material_type=None, headers=['Source Name', 'Characteristics[Organism]', 'Term Source REF', 'Term Accession Number', 'Characteristics[Source Type]', 'Characteristics[Cellline]', 'Characteristics[Genetic modfications]'])

s_Study_Title.txt: Found start node without original name: p4-s0-Empty Source Name-1-2
```

### LINKED ISSUE #6: AltamISA parsing fails with template "stem_cell_core_sc"
Similar to #5:

```
s_Study_Title.txt: Found annotated material/file without name: Material(type='Source Name', unique_name='p4-s0-Empty Source Name-1-1', name='', extract_label=None, characteristics=(Characteristics(name='Organism', value=[OntologyTermRef(name='Homo sapiens', accession='http://purl.bioontology.org/ontology/NCBITAXON/9606', ontology_name='NCBITAXON')], unit=None), Characteristics(name='Source Type', value=['hiPSC'], unit=None), Characteristics(name='Cellline', value=[''], unit=None), Characteristics(name='Genetic modifications', value=[''], unit=None)), comments=(), factor_values=(), material_type=None, headers=['Source Name', 'Characteristics[Organism]', 'Term Source REF', 'Term Accession Number', 'Characteristics[Source Type]', 'Characteristics[Cellline]', 'Characteristics[Genetic modifications]'])

s_Study_Title.txt: Found start node without original name: p4-s0-Empty Source Name-1-1

s_Study_Title.txt: Found annotated material/file without name: Material(type='Source Name', unique_name='p4-s0-Empty Source Name-1-2', name='', extract_label=None, characteristics=(Characteristics(name='Organism', value=[OntologyTermRef(name='Homo sapiens', accession='http://purl.bioontology.org/ontology/NCBITAXON/9606', ontology_name='NCBITAXON')], unit=None), Characteristics(name='Source Type', value=['hiPSC'], unit=None), Characteristics(name='Cellline', value=[''], unit=None), Characteristics(name='Genetic modifications', value=[''], unit=None)), comments=(), factor_values=(), material_type=None, headers=['Source Name', 'Characteristics[Organism]', 'Term Source REF', 'Term Accession Number', 'Characteristics[Source Type]', 'Characteristics[Cellline]', 'Characteristics[Genetic modifications]'])

s_Study_Title.txt: Found start node without original name: p4-s0-Empty Source Name-1-2
```

### PR DESCRIPTION
**Pull request by sellth**
to avoid validation errors during testing

fixes #5 & fixes #6

### CODE REVIEW

File: cubi_isa_templates/isatab-stem_cell_core_bulk/cookiecutter.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "sample_numbers": "0001,0002",
   "investigation_title": "{{cookiecutter.group}} - {{cookiecutter.study_title}}",
   "source_type": ["hiPSC", "tissue"],
-  "cellline": "",
+  "cellline": "CL123",
```
Comment (Nicolai-vKuegelgen) [inline cubi_isa_templates/isatab-stem_cell_core_bulk/cookiecutter.json:7]: please use "BIHiXXX-Y" if a default is needed

**Review by Nicolai-vKuegelgen — COMMENTED** [review]
If we do need to set a default please use this: "BIHiXXX-Y"
(There was intentionally no default defined, so that people don't retain a default that will be wrong in most cases),

**Comment by sellth** [comment]
Changed default value to match. @mikkonie do we actually need a default here?

**Comment by sellth** [comment]
Altamisa validation requires the Source Name column to be filled with something.

**Comment by mikkonie** [comment]
Yes, seems to me this is a good way to fix it.

**Pull request merged**